src/test/java/racingcar/domain/CarsTest.java

```diff
@@ -0,0 +1,29 @@
+package racingcar.domain;
+
+import org.junit.jupiter.api.Test;
+import racingcar.domain.validator.CarNameValidator;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CarsTest {
+
+    @Test
+    void 자동차_리스트에_들어가있는지_테스트() {
+        //given
+        CarNameValidator carNameValidator = new CarNameValidator();
+        Cars cars = new Cars("hello,I'm,bo,wook,^^", carNameValidator);
+
+        //when
+        List<Car> carList = cars.getCars();
+
+        //then
+        assertThat(carList.size()).isEqualTo(5);
+        assertThat(carList.get(0).getCarName()).isEqualTo("hello");
+        assertThat(carList.get(1).getCarName()).isEqualTo("I'm");
+        assertThat(carList.get(2).getCarName()).isEqualTo("bo");
+        assertThat(carList.get(3).getCarName()).isEqualTo("wook");
+        assertThat(carList.get(4).getCarName()).isEqualTo("^^");
```

Comment:
get함수를 여러 번 사용해서 검증하는 것은 가독성과 비용면에서 좋지 않다고 생각합니다.
```suggestion
        //then
        assertThat(carList)
                .extracting(Car::getCarName)
                .containsExactly("hello", "I'm", "bo", "wook", "^^");
```

또는 

```suggestion
        List<Car> expected = List.of(
        new Car("hello"),
        new Car("I'm"),
        new Car("bo"),
        new Car("wook"),
        new Car("^^")
    );

    //then
    assertThat(actual)
            .usingRecursiveComparison()
            .isEqualTo(expected);
```
이렇게 바꿔보시는 것을 추천드려요

Reply:
assertThat에 이런 기능이 있는 줄 몰랐네요! 알려주셔서 감사합니다.